src/vs/workbench/contrib/terminal/browser/terminal.ts

```diff
@@ -90,6 +91,29 @@ export const enum TerminalConnectionState {
 	Connected
 }
 
+export const enum TerminalTarget {
```

Comment:
Panel instead of TerminalView ?

Reply:
I think we're meant to move away from "panel" and to the more generic "view" since the "terminal panel" can be dragged into the side bar.